Gulpfile.js

```diff
@@ -0,0 +1,36 @@
+var gulp = require('gulp'),
+    autoprefix = require('gulp-autoprefixer'),
+    sass = require('gulp-sass'),
+    connect = require('gulp-connect'),
+    notify = require('gulp-notify'),
+    open = require('gulp-open'),
```

Comment:
Redefinition of 'open'.<br>'require' is not defined.<br>'open' is defined but never used.